src/platform/WebPlatformFunctions.ts

```diff
@@ -0,0 +1,54 @@
+import StripeEmitter = require('../StripeEmitter');
+import PlatformFunctions = require('./PlatformFunctions');
+
+/**
+ * Specializes WebPlatformFunctions using APIs available in Web workers.
+ */
+class WebPlatformFunctions extends PlatformFunctions {
+  /** @override */
+  getUname(): Promise<string | null> {
+    return Promise.resolve(null);
+  }
+
+  /** @override */
+  uuid4(): string {
```

Comment:
do we need a copy here?

Reply:
Oh good point, I missed this during the refactor